### ISSUE TITLE
should I finish the utils.load_mask?

### ISSUE THREAD

**Issue by buaaMars** [multimodallearning/pytorch-mask-rcnn#31]
If I want to use your excellent code to train mask r-cnn on **COCO**, do I need to finish the load_mask in utils?
As far as I am concerned, the unfinished load_mask is a severe problem which prevent trainer from taking the advantage of semantic segment ground truth.
Do I understand your code right?
Did you train your mask r-cnn model on COCO without the true mask, which resulted in a 5 points loss from the paper?

**Comment by buaaMars** [comment]
ok, I found the overrided load_mask() in CocoDataset. Problem solved!
